ssl.go

```diff
@@ -31,7 +31,7 @@ func ssl(o values) func(net.Conn) net.Conn {
 		// behavior is discouraged, and applications that need certificate
 		// validation should always use verify-ca or verify-full.
 		if sslrootcert, ok := o["sslrootcert"]; ok {
-			if _, err := os.Stat(sslrootcert); err == nil {
```

Comment:
This should stay since the `err` here is definitely not the same as the one in the return.

Reply:
Done.